app/src/main/java/com/android/calendar/alerts/AlertReceiver.java

```diff
@@ -163,9 +161,12 @@ public static void finishStartingService(Service service, int startId) {
     }
 
     private static PendingIntent createClickEventIntent(Context context, long eventId,
-            long startMillis, long endMillis, int notificationId) {
-        return createDismissAlarmsIntent(context, eventId, startMillis, endMillis, notificationId,
-                DismissAlarmsService.SHOW_ACTION);
+        long startMillis, long endMillis) {
+        Intent intent = AlertUtils.buildEventViewIntent(context, eventId, startMillis, endMillis);
```

Comment:
todo: after receiving the intent in `EventInfoActivity` we have to update the state of the alert from `STATE_FIRED` to `STATE_DISMISSED` to ensure that the notification will not be displayed again.

see https://developer.android.com/reference/android/provider/CalendarContract.CalendarAlertsColumns#STATE